js/values/Paper.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Construct the Paper object and set the keyword property
+ * @param keyword
+ * @constructor
+ */
+var Paper = function( keyword ) {
+	this.keyword = keyword || "";
+};
+
+/**
+ * Check whether a keyword is available
+ * @returns {boolean}
+ */
+Paper.prototype.hasKeyword = function() {
+	return ( typeof this.keyword !== "undefined" && this.keyword !== "" );
```

Comment:
This is redundant because of line 7. Furthermore, I rather have an `isEmpty` check here.